org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/capabilities/XMLCapabilityManager.java

```diff
@@ -153,9 +159,20 @@ public void initializeCapabilities() {
 		if (this.getClientCapabilities().isReferencesDynamicRegistrationSupported()) {
 			registerCapability(REFERENCES_ID, TEXT_DOCUMENT_REFERENCES);
 		}
+		if (this.getClientCapabilities().isDidChangeWatchedFilesRegistered()) {
+			registerWatchedFiles();
+		}
+		
 		syncDynamicCapabilitiesWithPreferences();
 	}
 
+	private void registerWatchedFiles() {
+		List<FileSystemWatcher> watchers = new ArrayList<>();
+		watchers.add(new FileSystemWatcher("**/*.xsd"));
```

Comment:
why not also `**/*.dtd`?

Reply:
DOMUtils provides constants for xsd and dtd extension file. I think we should have a constants which build this regexp in DOMUtils. I say that because there are other extension file like ent for dtd. And perhaps we should give the capability to add other extension file. See my comment https://github.com/angelozerr/lsp4xml/issues/380#issuecomment-494191582

Reply:
@angelozerr for now, should I just add `**/*.dtd` here, manually in the code?

Reply:
Do like this for the moment, but we need to change it in the future to give the capability to user to define custom file extension to watch.